File: in_Java/1304 Find N Unique Integers Sum up to Zero.java
class Solution {
    public int[] sumZero(int n) {
        int[] res = new int[n];
        int id = 0;
        int i = 1;
        while(n > 1){
            res[id] = i;
            res[id+1] = -i;
            id += 2;
            i++;
            n -= 2;
        }
        return res;
    }
}

File: in_Python/1304 Find N Unique Integers Sum up to Zero.py
class Solution(object):
    def sumZero(self, n):
        """
        :type n: int
        :rtype: List[int]
        """
        res = []
        m = n/2
        nv = 1
        while m != 0:
            res.append(nv)
            res.append(-nv)
            nv += 1
            m -= 1
        if n % 2 == 1:
            res.append(0)
        return res
        